README.md

```diff
@@ -0,0 +1,58 @@
+# Syncbot
```

Comment:
Good README ♥️ 

Could you please (may be in separate PR or this one) consider documenting the development env for this? It will be helpful for those who try to maintain / fix issues in future to understand how a change to the workflow can be tested.

Reply:
Created a separate PR for readme.

https://github.com/rhobs/syncbot/pull/2